motr/io_req.c

```diff
@@ -1573,14 +1573,24 @@ static int ioreq_dgmode_read(struct m0_op_io *ioo, bool rmw)
 	M0_ENTRY();
 	M0_PRE_EX(m0_op_io_invariant(ioo));
 
-	/*
-	 * If all devices are ONLINE, all requests return success.
-	 * In case of read before write, due to CROW, COB will not be present,
-	 * resulting into ENOENT error.
-	 */
+        /*
+         * Return immediately if all devices are ONLINE and all requests
+         * return success.
+         *
```

Comment:
Looks like indentation of the comment does not match with the rest of the file.

Otherwise, the patch looks good.